lbuild/module.py

```diff
@@ -236,6 +236,10 @@ def __getitem__(self, key: str):
         try:
             name = self.module.fill_partial_name(partial_name)
             key = ":".join(name)
+            find_first = (key[-1] == '*')
+            if key not in self.modules:
+                keys = [k for k in self.modules if k.startswith(key[:-1])]
+                if len(keys): key = keys[0];
```

Comment:
Please don't use this style. There is (almost) no need for semicolons in python :smile: